087-adopt-connect-health-endpoint.md

```diff
@@ -0,0 +1,84 @@
+# Adopt the Kafka Connect health check endpoint
+
+In a Kubernetes node, the kubelet component uses the configured liveness probe to know when to restart a container (i.e. deadlock).
+Instead, it uses the configured readiness probe to know when a container is ready to start accepting traffic (i.e. startup phase completed).
```

Comment:
The `Instead` does not seem to fit into how the text flows? Also, the test seems a but confusing with regards to startup probes. Maybe you could just focus that it is used to verify if the pod is ready to accept traffic?